docs/source/user_guide/faq.rst

```diff
@@ -62,3 +62,52 @@ lot longer (in the order of days)
 2. Koalas takes a different approach that might contradict Spark's API design principles, and those
 principles cannot be changed lightly given the large user base of Spark. A new, separate project
 provides an opportunity for us to experiment with new design principles.
+
+What is the list of APIs that are not planned to support in Koalas?
+-------------------------------------------------------------------
+
+Koalas doesn't support several APIs that may cause memory issues mostly due to the size of the data.
+
+For example, implementing and using `DataFrame.values` in Koalas can cause all data belonging to the
+
+DataFrame to be loaded into the driver's memory, causing memory errors like OOM.
+
+The following is a list of APIs that Koalas doesn't plan to support.
+
+- DataFrame
```

Comment:
Can we automatically generate this list and the APIs we didn't implement yet?

Reply:
okay let me try,

and IMO, i think we'd better separate the list into `Not yet implemented` and `Not will be implemented`.